iml-gui/crate/src/components/dropdown.rs

```diff
@@ -1,7 +1,7 @@
 use crate::{components::Placement, generated::css_classes::C};
 use seed::{prelude::*, virtual_dom::Attrs, Style, *};
 
-pub(crate) fn wrapper_view<T>(attrs: Attrs, placement: Placement, open: bool, children: impl View<T>) -> Node<T> {
+pub fn wrapper_view<T>(attrs: Attrs, placement: Placement, open: bool, children: impl View<T>) -> Node<T> {
```

Comment:
I think is it not used _yet_, but will be  soon.

Reply:
Already
https://github.com/whamcloud/integrated-manager-for-lustre/blob/master/iml-gui/crate/src/components/paging.rs#L167